src/gt4py/next/iterator/transforms/inline_lifts.py

```diff
@@ -138,7 +138,7 @@ class Flag(enum.IntEnum):
         #: when we see that it is not required.
         INLINE_LIFTED_ARGS = 16
 
-    predicate: Callable[[ir.Expr, bool], bool] = lambda _1, _2: True  # noqa: E731 [lambda-assignment]
+    predicate: Callable[[ir.Expr, bool], bool] = lambda _1, _2: True  # [lambda-assignment]
```

Comment:
Why is the `noqa` not longer needed? In any case, the comment with the error code should be removed.
```suggestion
    predicate: Callable[[ir.Expr, bool], bool] = lambda _1, _2: True
```